docs/command_reference.md

```diff
@@ -0,0 +1,279 @@
+```
+
+If you don't include the `--remote` flag, Notary deletes local cached content
+but will not delete data from the Notary server.
+
+## Change the passphrase for a key
+
+The Notary CLI client manages the keys used to sign the trusted collection. These keys are encrypted at rest.
+To list all the keys managed by the Notary CLI client, run:
+
+```bash
+$ notary key list
+```
+
+To change the passphrase used to encrypt one of the keys, run:
+
+```bash
+$ notary key passwd <key_id>
+```
+
+## Rotate keys
+
+If one of the private keys is compromised you can rotate that key, so that
+content that was signed with those keys stop being trusted.
+
+For keys that are kept offline and managed by the Notary CLI client, such the
+keys with the root, targets, and snapshot roles, you can rotate them with:
+
+```bash
+$ notary key rotate <GUN> <key_role>
+```
+
+The Notary CLI client generates a new key for the role you specified, and
+prompts you for a passphrase to encrypt it.
+Then you're prompted for the passphrase for the key you're rotating, and if it
+is correct, the Notary CLI client contacts the Notary server to publish the
+change.
+
+After a rotation, all previously existing keys for the specified role are replaced with the new key.
+
+You can also rotate keys that are stored in the Notary server, such as the keys
+with the snapshot or timestamp role. For that, run:
+
+```bash
+$ notary key rotate <GUN> <key_role> -r
+```
+
+## Importing and exporting keys
+
+Notary can import keys that are already in a PEM format:
+```bash
+$ notary key import <pemfile> --role <key_role> --gun <key_gun>
+```
+
+If no `--role` or `--gun` is given, notary will assume that the key is to be used for a delegation role.
+Moreover, it's possible for notary to import multiple keys contained in one PEM file, each separated into separate PEM blocks.
+
+For each key it attempts to import, notary will prompt for a passphrase so that the key can be encrypted at rest.
+
+Notary can also export all of its encrypted keys, or individually by key ID or GUN:
+```bash
+# export all my notary keys to a file
+$ notary key export -o exported_keys.pem
+
+# export a single key by ID
+$ notary key export --key <keyID> -o exported_keys.pem
+
+# export multiple keys by ID
+$ notary key export --key <keyID1> --key <keyID2> -o exported_keys.pem
+
+# export all keys for a GUN
+$ notary key export --gun <GUN> -o exported_keys.pem
+
+# export all keys for multiple GUNs
+$ notary key export --gun <GUN1> --gun <GUN2> -o exported_keys.pem
+```
+When exporting multiple keys, all keys are outputted to a single PEM file in individual blocks. If the output flag `-o` is omitted, the PEM blocks are outputted to STDOUT.
+
+## Manage keys for delegation roles
+
+To delegate content signing to other users without sharing the targets key, retrieve a x509 certificate for that user and run:
+
+```bash
+$ notary delegation add -p <GUN> targets/<role> --all-paths user.pem user2.pem
```

Comment:
I think it's the common use case so I used that, but I'll make a note about it and `--paths`